assignments/function-conversion.js

```diff
@@ -4,27 +4,47 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+ 
+  let myFunction=()=>{console.log("Function was invoked!");};
+  myFunction();
+
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+  let anotherFunction=(param)=>{return param;};
```

Comment:
you can also exclude the return statement and the curly braces if the arrow function is a one liner